recipes/gdal/post_3.5.0/conanfile.py

```diff
@@ -118,6 +120,8 @@ class GdalConan(ConanFile):
         "with_proj": True,
         "with_qhull": True,
         "with_sqlite3": True,
+        "with_spatialite": True,
+        "with_rasterlite2": False,
```

Comment:
Do not enable by default please. These libs are quite confidentials.

Reply:
OK, done. I also disabled sqlite3, because spatialite is tightly linked with it. Most likely, a user needing sqlite3 will also need spatialite.